docs/UserGuide.md

```diff
@@ -12,58 +15,69 @@ StudyTracker is a **desktop app that lets you save your favourite study spots, o
+   1. **`exit`** : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command.
+1. Refer to the [Features](#features) below for more commands and features!
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command Syntax
 
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Throughout the User Guide, you may see commands to enter into the bot. Here is how to read the command format:**<br>
+**:information_source: Throughout the User Guide, you'll see commands to enter into StudyTracker.
+Here is how to read the command format:**<br>
 
-* The first word of every line *always* specifies the **command word**. Command words are single words. Commands may have aliases, which are shortened versions of the command word. <br>
-  e.g. `add` has the alias `new`.
+* The first word of each command *always* specifies the **command word**.
+  Commands words may be user-created aliases, which are detailed further in [Aliases](#setting-command-aliases--alias-unalias). <br>
 
-* The subsequent blocks are known as **parameters**. Each parameter has a delimiter. <br>
+* Commands may require **flags**.
+  Flags are represented by a dash, and a sequence of characters. <br>
+  e.g. the `list` command has a `-f` flag to list only favourites.
+
+* The subsequent blocks are known as **parameters**.
+  Each parameter may have a delimiter. <br>
   e.g. the `NAME` parameter has a delimiter `n` and the `AMENITY` parameter has a delimiter `m`.
 
-* Slashes are required for user input. Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+* Slashes are required for parameter input.
+  Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME*`, `NAME` is a parameter which can be used as `add n/COM1 Basement`.
 
 * Parameters with an asterisk `*` are required while those without an asterisk are optional.<br>
   e.g `n/NAME* m/AMENITY...` can be used as `n/COM1 Basement m/wifi` or as `n/COM1 Basement`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
+* Parameters with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `t/TAG...​` can be used as ` ` (i.e. 0 times), `t/noisy`, `t/mosquitos t/sunny` etc.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME* a/ADDRESS`, `a/ADDRESS n/NAME*` is also acceptable.
 
 * If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `a/COM1 a/CLB`, only `a/CLB` will be taken.
+  e.g. if you specify `n/COM1 n/CLB`, only `n/CLB` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
```

Comment:
For line 86: Can change to 

* `add n/PC Commons a/UTown t/crowded m/wifi r/4` //tag doesn't work with spaces
* `add n/COM2 Basement r/3` (optional arguments are not required) // only name and rating compulsory
* `add n/TR3 a/Yusof Ishak House m/wifi m/aircon` (multiple amenities are acceptable) //changed amenities to those we currently have, and I'm not sure why it's `new` not `add` for the prev ver

Can also add more variations if you want, e.g. multiple tags acceptable